webui/react/src/pages/ExperimentDetails/ExperimentSingleTrialTabs.tsx

```diff
@@ -146,7 +158,7 @@ const ExperimentSingleTrialTabs: React.FC<Props> = ({
     }
   }, [canceler.signal, onTrialUpdate, trialId]);
 
-  const { stopPolling } = usePolling(fetchTrialDetails, { rerunOnNewFn: true });
+  const { stopPolling } = usePolling(fetchTrialData, { rerunOnNewFn: true });
```

Comment:
looks like its not polling? is it expected? 

https://github.com/determined-ai/determined/assets/109113616/f198dd0f-4a70-4796-be9f-f1aa169fdd2f

Reply:
fixed